src/luarocks/build.lua

```diff
@@ -389,17 +390,18 @@ end
 -- @return boolean or (nil, string, exitcode): True if build was successful; nil and an
 -- error message otherwise. exitcode is optionally returned.
 function build.command(flags, name, version)
+   local deps_mode = deps.get_deps_mode(flags)
```

Comment:
With this line you can also pass `deps_mode` directly to `manif.check_dependencies` below.